modules/logging/formattedencoder.go

```diff
@@ -17,7 +15,7 @@ func init() {
 	caddy.RegisterModule(FormattedEncoder{})
 }
 
-const commonLogFormat = `{http.common_log}`
+const commonLogFormat = `{common_log}`
```

Comment:
Where is this used?

Reply:
It's used in the `UnmarshalCaddyfile` as fallback if no template is given. The JSON config doesn't have the same fallback. Should I implement the same?

Reply:
Ah, no, I think this is fine. Thanks